homeassistant/components/evohome/manifest.json

```diff
@@ -5,5 +5,6 @@
   "documentation": "https://www.home-assistant.io/integrations/evohome",
   "iot_class": "cloud_polling",
   "loggers": ["evohomeasync", "evohomeasync2"],
-  "requirements": ["evohome-async==0.4.20"]
+  "requirements": ["evohome-async==0.4.20"],
+  "single_config_entry": true
```

Comment:
Why is this limited to a single entry?

Reply:
There were too much moving parts currently, I think this requires a bigger refactor at the moment